datastore/common/common.go

```diff
@@ -22,3 +129,14 @@ func Getenv(key string, defaultValue string) string {
 func Tstamp2float64Secs(tstamp *timestamppb.Timestamp) float64 {
 	return float64(tstamp.GetSeconds()) + float64(tstamp.GetNanos())/1e9
 }
+
+// GetValidTimeRange returns the current valid time range as (lo time, hi time).
+func GetValidTimeRange() (time.Time, time.Time) {
+	if dynamicTime { // Case 1: dynamic time, i.e. relative to current time
+		now := time.Now().Unix()
+		return time.Unix(now - loTimeSecs, 0), time.Unix(now - hiTimeSecs, 0)
```

Comment:
If `hiTimeSecs=0`, this rounds the current time done when calculating the upper limit, correct? This means current observations are very likely to be cut off, which is probably not what we want. 

Maybe we should always have a couple of seconds leeway?

Reply:
Agreed, but then I think that leeway should be configured through the environment variable `HITIME` (i.e. setting it to `-2` for a leeway of 2 secs etc.). Fine-tuning a service can then be done more easily, i.e. without having to recompile. I guess one could also set the leeway to something larger, like one hour, but then the store would allow observations that long into the future, which is probably fine? This would anyway indicate a problem on the ingestion side.

Reply:
That should be fine, but we should set the default to something like `-2` to avoid unexplained behaviour.

Reply:
Agreed, will fix